app/controllers/admin_pages_controller.rb

```diff
@@ -1,16 +1,16 @@
 class AdminPagesController < ApplicationController
-  before_action :authorize_admin, :get_users
+  before_action :authorize_admin, :get_users_for_admin
+  Pagy::DEFAULT[:items] = Settings.digits.size_of_admin_page
 
   def index
-    check_no_params = User::CREATE_ATTRS.all? do |key|
-      params[key].nil?
-    end
-    check_no_params ? get_users : search
+    @search = User.ransack(params[:q])
```

Comment:
thế thì em if else
nếu có params thì search
còn k thì trả về toàn bộ danh sách
chứ e để ntn nó phải query 2 lần

Reply:
em đã fix ạ